src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java

```diff
@@ -153,7 +155,7 @@ MessageML parse(String message, String entityJson, String version) throws Invali
     String expandedMessage;
 
     if (StringUtils.isBlank(message)) {
-     throw new InvalidInputException("Error parsing message: the message cannot be null or empty");
+      throw new InvalidInputException("Error parsing message: the message cannot be null or empty");
     }
 
     if (StringUtils.isNotBlank(entityJson)) {
```

Comment:
would it be cleaner to set the ENTITY_JSON_SIZE value here? since we can check if the object node has any children (to avoid the > 2 magic)

Reply:
unfortunately i can't check that before because at this point the message is not parsed yet and for entities like: mentions, cashtags we are going to fill the entityJson. So it could happen that here we have size 0 but at the end is not empty anymore.